modules/virt-creating-storage-class-csi-driver.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+// * virt/storage/virt-configuring-local-storage-with-hpp.adoc
```

Comment:
The [numbering is wrong](https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.html#virt-creating-storage-class-csi-driver_virt-configuring-local-storage-for-vms) in the procedure. It is wrong in the [live docs](https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.html#virt-creating-storage-class-csi-driver_virt-configuring-local-storage-for-vms), not from anything you did. Perhaps you can fix it? Maybe remove the blank line 37 or add a `+`?